mgosimplekv/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package mgosimplekv_test
+
+import (
+	"testing"
+
+	jujutesting "github.com/juju/testing"
```

Comment:
what about using quicktest instead ?

Reply:
I'd like to but mongodb. Also I'd like to keep the tests from candid as much as possible initially. Hopefully in a later PR.